docs/rules/0133/method-signature.md

```diff
@@ -0,0 +1,84 @@
+---
+rule:
+  aip: 133
+  name: [core, '0133', method-signature]
+  summary: |
+    Create RPCs should annotate an appropriate method signature.
+permalink: /133/method-signature
+redirect_from:
+  - /0133/method-signature
+---
+
+# Create methods: Method signature
+
+This rule enforces that all `Create` standard methods have a
+`google.api.method_signature` annotation with an appropirate value, as mandated
+in [AIP-133][].
+
+## Details
+
+This rule looks at any method beginning with `Create`, and complains if the
+`google.api.method_signature` annotation is missing, or if it is set to an
+incorrect value. Additional method signatures, if present, are ignored.
+
+The correct value is `"parent,{resource},{resource}_id"` if the `{resource}_id`
+field exists, and `"parent,{resource}"` otherwise.
+
+## Examples
+
+**Incorrect** code for this rule:
+
+```proto
+// Incorrect.
+rpc CreateBook(CreateBookRequest) returns (Book) {
+  // A google.api.method_signature annotation should be present.
+}
+```
+
+```proto
+// Incorrect.
+rpc CreateBook(CreateBookRequest) returns (Book) {
+  // Should be "parent,book" or "parent,book,book_id", depending on whether
+  // a "book_id" field exists.
+  option (google.api.method_signature) = "publisher,book";
+}
+```
+
+**Correct** code for this rule:
+
+If the `book_id` field does not exist:
+
+```proto
+// Correct.
+rpc CreateBook(CreateBookRequest) returns (Book) {
+  option (google.api.method_signature) = "parent,book";
+}
+```
+
+If the `book_id` field exists:
```

Comment:
Maybe it is better to give an example of the `Book` message with the `book_id` field.